releasenotes/notes/add-grover-primitives-10f81efdba93703d.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+
+    :class:`~.Grover` supports the primitives and can use :class:`~.Sampler` to
+    calculate the results. Accordingly, ``quantum instance`` in :class:`~.Grover`
```

Comment:
The bit about deprecation I think should be split out of features and into a deprecation section as this is where other pending deprecation information is being put in other algo release notes